includes/admin/class-amp-post-meta-box.php

```diff
@@ -422,4 +429,79 @@ public function preview_post_link( $link ) {
 
 		return $link;
 	}
+
+	/**
+	 * Add a REST API field to display the AMP status on AMP supported post types.
+	 *
+	 * @return void
+	 */
+	public function add_rest_api_fields() {
+		register_rest_field(
+			AMP_Post_Type_Support::get_post_types_for_rest_api(),
```

Comment:
Shouldn't this rather be just simply:

```suggestion
			'post',
```

I'm not seeing a way to register a field for specific post types.

Reply:
Also asked the same below.

Reply:
Setting this to `post` means that the field will always be displayed on the `post` object. My intention here was to not register the REST field if the `post` content type was not marked as supported on the AMP settings page. I'll update this  to reflect that intention.

Reply:
Addressed in https://github.com/ampproject/amp-wp/pull/4689/commits/02625fb09eea381c65a4d1611cdbff77fef71bb7.